app/Resources/views/podcast-episode.html.twig

```diff
@@ -1,3 +1,5 @@
+{% set criticalCss = 'post' %}
```

Comment:
Might want to use a custom type as it is a little different (image + media player)?